letsencrypt/client/client.py

```diff
@@ -43,7 +44,6 @@ class Client(object):
     """
     zope.interface.implements(interfaces.IAuthenticator)
 
-    Key = collections.namedtuple("Key", "file pem")
```

Comment:
Any particular reason why this had to be moved to `le_util`? To keep the code consistent maybe below `CSR` should be moved as well?

Also, there is a lot of documentation in the code that refers to `letsencrypt.client.Client.Key` that hasn't been updated accordingly, `grep` to the rescue.